context_test.go

```diff
@@ -50,6 +52,16 @@ func TestGetCorrelationID(t *testing.T) {
 	assert.Equal(t, "testing-id", id)
 }
 
+func TestAppendNewRelicToClient(t *testing.T) {
+	echoCtx, _, _ := getEchoTestCtx()
+	ctx := NewContext(echoCtx, stubNewRelicApp(), &Logger{}, "correlationid", false, "build-1.2.3")
+	//check it uses the same client
+	const timeout = time.Second * 10
+	client := &http.Client{Timeout: timeout}
```

Comment:
Should we filling in a default timeout here ?
Maybe we should consider using some default transport layer timeouts too ?